database/tables_creation.sql

```diff
@@ -0,0 +1,170 @@
+-- Table: onboarding_schema.users
+
+-- DROP TABLE IF EXISTS onboarding_schema.users;
+
+CREATE TABLE IF NOT EXISTS onboarding_schema.users
+(
+    id integer NOT NULL,
+    name character varying COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT users_pkey PRIMARY KEY (id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS onboarding_schema.users
+    OWNER to postgres;
+
+-- Table: onboarding_schema.items
+
+-- DROP TABLE IF EXISTS onboarding_schema.items;
+
+CREATE TABLE IF NOT EXISTS onboarding_schema.items
+(
+    id integer NOT NULL,
+    name character varying COLLATE pg_catalog."default" NOT NULL,
+    CONSTRAINT items_pkey PRIMARY KEY (id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS onboarding_schema.items
+    OWNER to postgres;
+
+-- Table: onboarding_schema.comments
+
+-- DROP TABLE IF EXISTS onboarding_schema.comments;
+
+CREATE TABLE IF NOT EXISTS onboarding_schema.comments
+(
+    user_sender integer NOT NULL,
```

Comment:
No need to change now, but just for you to know. We usually add a suffix (id) to the foreign keys, for example: 
user_sender_id, user_receiver_id